### DISCUSSION TITLE
error when building SYCL for mips64

### DISCUSSION THREAD

**Discussion by bananAshkar** [intel/llvm#11741] [Q&A]
I am trying to build SYCL for mips64 , I tried with running this command :
python3 ./llvm/buildbot/configure.py -t Release --cmake-gen "Ninja" --host-target=Mips

then I run :
python3 ./llvm/buildbot/compile.py

and I started to see errors like that :
fatal error: 'climits' file not found
   14 | #include <climits>
error: unknown target triple 'unknown'

**Comment by bader** [comment]
I should say that we don't test mips64 as host system for DPC++ in our CI, so it's possible that something might be broken.

> I am trying to build SYCL for mips64

What system to do build SYCL **on**? Is it x86 or mips?
Could you attach the full output of both configure and compile scripts? It's unclear which compiler produces that error: host compiler that builds DPC++ compiler to DPC++ compiler itself while building device libraries.

**Reply by bananAshkar** [reply]
I am building on x86

the full output of the configure is : 
[ConfigureOutput.txt](https://github.com/intel/llvm/files/13245207/ConfigureOutput.txt)

the full output of the compile is :
[CompileOutput.txt](https://github.com/intel/llvm/files/13245323/CompileOutput.txt)

**Reply by bader** [reply]
Sorry, for the delay.

The build of device library is failing. We build DPC++ compiler first and then use it to build "device library" written in SYCL. DPC++ can't find MIPS version of libstdc++ (i.e. standard C++ library). Do you have it on your system? If no, could you try to install it and check again?
